src/components/CollaborationBar/Collaborators/Collaborators.tsx

```diff
@@ -91,6 +108,9 @@ export function Collaborators() {
           />
         }
       >
+        {presentingUser && (
+          <ToolbarAvatarPresenter member={{ userId: presentingUser }} />
+        )}
         {orderedMembers.map((m) => (
```

Comment:
This makes `moreButton` provided by `ToolbarAvatarGroup` not work properly. It could work fine if presenter user is part of `orderedMembers` and is first entry there.